zipkin-common/src/main/scala/com/twitter/zipkin/storage/SpanStore.scala

```diff
@@ -149,6 +149,23 @@ class InMemorySpanStore extends SpanStore with CollectAnnotationQueries {
       .toList
   }
 
+  override protected def getTraceIdsByDuration(
+    serviceName: String,
```

Comment:
I will add tests tonight or tomorrow, basically load 2 traces. One has root
span with service A calling child span with service B. The other is visa
versa.

I suspect the intended behavior is that one returns, not both!